packages/css-framework/src/components/_transfer.scss

```diff
@@ -0,0 +1,59 @@
+.rn-transfer {
+  .column-left,
```

Comment:
Can `column-left` and `column-right` be replaced with one generic class?

Reply:
See, this is what comes of having a 3 week gap in between development. This used to be needed but I refactored it out in the previous sprint and totally forgot about it by the time I picked it up again in this one :-/